kong/api/endpoints.lua

```diff
@@ -172,17 +170,17 @@ local function get_collection_endpoint(schema, foreign_schema, foreign_field_nam
       return helpers.responses.send_HTTP_NOT_FOUND()
     end
 
-    local size, err = get_page_size(self.args.uri)
+    local id = { id = foreign_entity.id }
```

Comment:
This is a bit nitpickish, but this variable name seems misleading. I suggest renaming to `pk`.